frontend/src/app/order-table/order-table.component.ts

```diff
@@ -40,4 +40,10 @@ export class OrderTableComponent implements OnInit {
     this.router.routeReuseStrategy.shouldReuseRoute = function() { return false; };
     this.router.navigate([this.router.url])
   }
+
+  finishOrder(order : number): void{ 
+    this.orderControllerService.finishOrder(order).subscribe();
+    this.router.routeReuseStrategy.shouldReuseRoute = function() { return false; };
+    this.router.navigate([this.router.url])
```

Comment:
A semicolon might be missing at the end of the line.

Reply:
Right, fixed. I added a little space between 2 buttons as well.